langlib/lang.compilertest/src/main/ballerina/src/lang.compilertest/compilertest.bal

```diff
@@ -0,0 +1,6 @@
+
+public function assertValueEqual(anydata expected, anydata actual) = external;
```

Comment:
Please add Ballerina documentations.

Btw, We can improve the API. Let's do it next iteration.